satip/utils.py

```diff
@@ -816,17 +820,19 @@ def create_markdown_table(table_info: dict, index_name: str = "Id") -> str:
     return md_str
 
 
-def save_to_zarr_to_s3(dataset: xr.Dataset, filename: str):
-    """Save xarray to netcdf in s3
+def save_to_zarr_to_s3(dataset: xr.Dataset, filename: str, backend: str = "s3"):
```

Comment:
Could this be changed to 
```suggestion
def save_to_zarr_to_backend(dataset: xr.Dataset, filename: str, backend: str = "s3"):
```

Reply:
This will also then need some other changes in the rest of the codebase, as this function is called a few times

Reply:
> This will also then need some other changes in the rest of the codebase, as this function is called a few times

Done!